src/components/MenuButton/__snapshots__/MenuButton.test.tsx.snap

```diff
@@ -157,6 +163,7 @@ exports[`<MenuButton /> with all props matches its snapshot 1`] = `
         Object {
           "data": Object {
             "yamUIIcon": undefined,
+            "yamUIIllustration": undefined,
```

Comment:
Probably wouldn't hurt to add an illustration to one of these list items

Reply:
Added an Icon to one menu item and an Illustration to another for these tests.